importer_client/python/tools/timesketch_importer.py

```diff
@@ -143,10 +143,30 @@ def upload_file(
         timeline = streamer.timeline
```

Comment:
To make it easier for yourself, you could use this `timeline` object and use its `run_analyzers()` function instead of directly speaking to the API.
See the function of the timeline object in the API client here: https://github.com/google/timesketch/blob/master/api_client/python/timesketch_api_client/timeline.py#L225C9-L225C22

 What do you think?

Reply:
This `timeline` object also returns the ID of the timeline that was just uploaded! (`timeline.id`)